kubernetes/cray-bos/values.yaml.in

```diff
@@ -34,10 +37,11 @@ cray-service:
     cray-bos:
       name: cray-bos
       image:
-        repository: cray/cray-bos
+        repository: artifactory.algol60.net/csm-docker/stable/cray-bos
+        # tag defaults to chart appVersion
       ports:
       - name: http
-        containerPort: 8000
+        containerPort: 9000
```

Comment:
Why are we changing the containerPort?

Reply:
Master branch is apparently lacking non-root mods. We should open a ticket to track to see if there is anything further missing, or submit a PR 1.2 --- merge --> master to capture everything else that may be missing.

Reply:
I created CASMCMS-7696 to do the non-root audit. 
Why is port 8000 considered a non-root change? Typically only ports 1024 and under are restricted to root for the Linux kernel.

Reply:
uwsgi cannot bind to port 8000 when launched as non-root.